Index: bc.gpr
===================================================================
--- bc.gpr	(revision 1511)
+++ bc.gpr	(working copy)
@@ -466,7 +466,7 @@
          --  required interface units to ./include.
          null;
    end case;
-   for Library_Options use External_As_List ("LDFLAGS", " ");
+   --  for Library_Options use External_As_List ("LDFLAGS", " ");
 
    for Languages use ("ada");
 
